### ISSUE TITLE
Investigate: Reduce to only using one grcp port and one http port

### ISSUE THREAD

**Issue by Laremere** [googleforgames/open-match#991]
Given that multiple grpc services can be hosted on the same port, and same with http, it seems like we could get away with Open Match only using two ports across all services.  http(s) should use the standard ports, and grpc should use whatever the standard is there.  They might even be able to use the same port.

This would reduce the complexity across the stack: Simply use the same port numbers everywhere.  Different grpc services would translate to different k8s services anyways, so that would all work.  In a mini-match case, all the services should be on the same grpc server anyways.

**Comment by yfei1** [comment]
Some thoughts about this change. I definitely think using one port for all services can greatly reduce the complexity of our configurations, but our current debugging experience actually greatly benefits from using different ports for each service. For example, if we see some errors like `failed to handshake with xxxx:50503` from our director, we will then realize the call failed to fetch data from the mmlogic service. It would be a headache to debug the connection if we directly unify the ports without any improvements on logging.

**Comment by Laremere** [comment]
Good point.  We should fix the errors to have more details.

**Comment by sawagh** [comment]
@Laremere - looks like the attempt to fix this was closed. Can you summarize the challenge you hit and whether we will still address this issue?

**Comment by Laremere** [comment]
I closed it because the tests were failing and I didn't want to spend the time debugging them instead of making progress on scale tests.  I thought there was an easier workaround which is why I closed it (it wasn't easier, but the second one I found was).

We should still do this, it's just a matter of taking the time to do it right.
